src/components/EditableText/EditableText.test.tsx

```diff
@@ -99,13 +104,13 @@ describe('<EditableText />', () => {
 
       describe('and text is changed', () => {
         beforeEach(() => {
-          const onChange: (text: string) => void = component.find(TextField).prop('onChange') as (text: string) => void;
+          const onChange = component.find(TextField).prop('onChange') as ChangeHandler;
```

Comment:
i think line 95 above this was necessary to test the "else" test case for focusing textfield, you'd probably have to do the same to get full coverage on the Clickable focus